playbooks/robusta_playbooks/bash_enrichments.py

```
@@ -40,3 +40,39 @@ def node_bash_enricher(event: NodeEvent, params: BashParams):
     block_list.append(MarkdownBlock(f"Command results for *{params.bash_command}:*"))
     block_list.append(MarkdownBlock(exec_result))
     event.add_enrichment(block_list)
+
+
+@action
+def generic_bash_enricher(event: ExecutionBaseEvent, params):
```

Ruff (0.12.2)

Undefined name `ExecutionBaseEvent`

(F821)